README.md

```diff
@@ -46,15 +46,15 @@ Apache Commons Collections
 [![Build Status](https://travis-ci.org/apache/commons-collections.svg)](https://travis-ci.org/apache/commons-collections)
 [![Coverage Status](https://coveralls.io/repos/apache/commons-collections/badge.svg)](https://coveralls.io/r/apache/commons-collections)
 [![Maven Central](https://maven-badges.herokuapp.com/maven-central/org.apache.commons/commons-collections4/badge.svg)](https://maven-badges.herokuapp.com/maven-central/org.apache.commons/commons-collections4/)
-[![Javadocs](https://javadoc.io/badge/org.apache.commons/commons-collections4/4.4.svg)](https://javadoc.io/doc/org.apache.commons/commons-collections4/4.4)
+[![Javadocs](https://javadoc.io/badge/org.apache.commons/commons-collections4/4.4.svg)](https://commons.apache.org/proper/commons-collections/javadocs/api-4.4/)
 
 The Apache Commons Collections package contains types that extend and augment the Java Collections Framework.
 
 Documentation
 -------------
 
-More information can be found on the [Apache Commons Collections homepage](https://commons.apache.org/proper/commons-collections).
-The [Javadoc](https://commons.apache.org/proper/commons-collections/apidocs) can be browsed.
+More information can be found on the [Apache Commons Collections homepage](https://commons.apache.org/proper/commons-collections/).
+The [Javadoc](https://commons.apache.org/proper/commons-collections/apidocs/) can be browsed.
```

Comment:
I believe this file is generated automatically by the commons plug-ins.

Reply:
File has been rollbacked.